torchvision/tv_tensors/_dataset_wrapper.py

```diff
@@ -198,8 +199,19 @@ def __getitem__(self, idx):
     def __len__(self):
         return len(self._dataset)
 
+    # TODO: maybe we should use __getstate__ and __setstate__ instead of __reduce__, as recommended in the docs.
```

Comment:
See just [above this link](https://docs.python.org/3/library/pickle.html#object.__reduce__)

Reply:
We can try, but I think it is not possible. The "state" in `__get_state__` and `__set_state__` is the second return value of `__reduce__` below. And while we can recreate a `VisionDatasetTVTensorWrapper` from them, `pickle` does not know how to create the dynamic type. I'll give it a shot.